src/main/java/fi/vm/sade/varda/rekisterointi/configuration/HakijaWebSecurityConfig.java

```diff
@@ -109,39 +118,50 @@ public AuthenticationProvider hakijaAuthenticationProvider() {
         return authenticationProvider;
     }
 
-    private static class ShibbolethAuthenticationFilter extends AbstractAuthenticationProcessingFilter {
+    private static class HakijaAuthenticationFilter extends AbstractAuthenticationProcessingFilter {
```

Comment:
Does this really need to be private and static?

Reply:
i dont know but it was previous code and works